hoomd_organics/base/simulation.py

```diff
@@ -566,7 +795,29 @@ def run_NVT(
         thermalize_particles=True,
         write_at_start=True,
     ):
-        """"""
+        """
+        Run the simulation in NVT dynamics.
```

Comment:
I think it would be more correct to say "Run the simulation in the NVT ensemble". The dynamics thing really only applies to the Langevin method